frontend/util.lua

```diff
@@ -859,7 +859,7 @@ end
 function util.diskUsage(dir)
     -- safe way of testing df & awk
     local function doCommand(d)
-        local handle = io.popen("df -k " .. d .. " 2>&1 | awk '$3 ~ /[0-9]+/ { print $2,$3,$4 }' 2>&1 || echo ::ERROR::")
+        local handle = io.popen("df -k " .. d .. " 2>&1 | awk '$3 ~ /[0-9]+/ { print $2,$3,$4 }' 2>&1 | grep -v 'ERROR: ld.so' || echo ::ERROR::")
```

Comment:
Wouldn't 'ignored' or something make more sense?

Reply:
or redirect errors to `/dev/null` in `df` output

Reply:
Redirecting is probably the cleanest reasonable solution. Though I had to do the same with awk, as it's launched with the same LD_PRELOAD variable.

Reply:
FWIW, we could also filter out stuff from the env via `env`, but void'ing stderr is more generic ;).